### ISSUE TITLE
Have fetching and serving as two separate tasks 

### ISSUE THREAD

**Issue by vincentorback** [tictail/gulp-tictail#9]
Hey guys!
It would be sweet if the task stored a local `store.json` file for offline development so you don’t have to re-fetch the store data every time. So maybe have to separate tasks, one to fetch and one to serve.

Thanks!

**Comment by kalasjocke** [comment]
@vincentorback I totally agree with you. Do you think it would make sense to ship some kind of binary with this tool to do tasks like that (`$ tt fetch`), or would it be to the developer to add the `fetch` task to their own Gulpfile?

**Comment by vincentorback** [comment]
I was first thinking something like this:
```javascript
var storeId = 'pf';

gulp.task('fetch', function() {
    tictail.fetch({
        store_id: storeId
    });
});

gulp.task('serve', function() {
    tictail.serve({
        store_id: storeId
    });
});
```

But that might be too much to ask from developers who doesn’t care about this feature.
Maybe you can disable fetch as an option?
```javascript
// Use this one for fetching
gulp.task('serve', function() {
    tictail.serve({
        store_id: storeId
    });
});

// Use this one for serving
gulp.task('serve', function() {
    tictail.serve({
        store_id: storeId,
        fetch: false
    });
});
```

**Comment by vincentorback** [comment]
But if you have more use cases when you need a binary tool like `tt` that’d be nice as well! :smiley_cat:

**Comment by ptz0n** [comment]
@vincentorback Great idea! Making the API cache support file storage (in addition to memory) will also do the trick.

Also having a separate fetch task would be convenient for this use case.
